### ISSUE TITLE
Crash when switching from Second Space

### ISSUE THREAD

**Issue by ElDavoo** [TeamNewPipe/NewPipe#8105]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.22.1

### Steps to reproduce the bug

1. Switch from Second Space to Primary Space

### Expected behavior

Not crashing

### Actual behavior

Crash

## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ IT
* __Content Language:__ it-IT
* __App Language:__ it_IT
* __Service:__ none
* __Version:__ 0.22.1
* __OS:__ Linux Android 11 - 30
<details><summary><b>Crash log </b></summary><p>

```
java.lang.IllegalStateException: Not allowed to start service Intent { cmp=org.schabi.newpipe/.CheckForNewAppVersion }: app is in background uid UidRecord{7ffd29c u0a326 CAC  bg:+1h39m25s620ms idle procs:1 seq(0,0,0)}
	at android.app.ContextImpl.startServiceCommon(Unknown Source:112)
	at android.app.ContextImpl.startService(Unknown Source:6)
	at android.content.ContextWrapper.startService(Unknown Source:2)
	at org.schabi.newpipe.CheckForNewAppVersion.startNewVersionCheckService(CheckForNewAppVersion.java:250)
	at org.schabi.newpipe.MainActivity.onPostCreate(MainActivity.java:180)
	at android.app.Instrumentation.callActivityOnPostCreate(Unknown Source:0)
	at android.app.ActivityThread.handleStartActivity(Unknown Source:111)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(Unknown Source:140)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(Unknown Source:10)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(Unknown Source:25)
	at android.app.servertransaction.TransactionExecutor.execute(Unknown Source:76)
	at android.app.ClientTransactionHandler.executeTransaction(Unknown Source:7)
	at android.app.ActivityThread.handleRelaunchActivityLocally(Unknown Source:72)
	at android.app.ActivityThread.access$3500(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(Unknown Source:52)
	at android.os.Handler.dispatchMessage(Unknown Source:19)
	at android.os.Looper.loop(Unknown Source:250)
	at android.app.ActivityThread.main(Unknown Source:134)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(Unknown Source:11)
	at com.android.internal.os.ZygoteInit.main(Unknown Source:313)

```
</details>
<hr>

### Affected Android/Custom ROM version

MIUI 12.5 (Android 11)

### Affected device model

Xiaomi Mi 10T Pro

### Additional information

MIUI's "Second Space" is a feature that allows to have two separate spaces in one phone. It's just like having two phones: everything is separate, including settings, installed apps... It's just like a multi-user feature but more powerful.

However, sometimes, when switching from Second Space to the primary one, the crash happens. Note that newpipe is only installed in the primary space and it was not running at time of crash.

**Comment by TacoTheDank** [comment]
Duplicate of #7397 

Fixed with #7975

**Comment by ElDavoo** [comment]
ok, sorry for the dupe
